kik_unofficial/utilities/cryptographic_utilities.py

```diff
@@ -24,24 +24,20 @@ def make_kik_timestamp():
         i1 = (-16777216 & j) >> 24
         i2 = (16711680 & j) >> 16
         i3 = (65280 & j) >> 8
-    
+
         j2 = (30 & i1) ^ i2 ^ i3
         j3 = (224 & j) >> 5
         j4 = -255 & j
-    
-        if j2 % 4 == 0:
-            j3 = j3 // 3 * 3
-        else:
-            j3 = j3 // 2 * 2
-        
+
+        j3 = j3 // 3 * 3 if j2 % 4 == 0 else j3 // 2 * 2
         return j4 | (j3 << 5) | j2
 
     @staticmethod
     def key_from_password(username, password):
         # kik's secret algorithm for encrypting passwords
         # relevant source file: classes1\kik\android\chat\fragment\KikLoginFragmentAbstract.java
         sha1_password = binascii.hexlify(hashlib.sha1(password.encode('UTF-8')).digest())
-        salt = username.lower() + "niCRwL7isZHny24qgLvy"
```

Comment:
Actually I think the previous form is easier to understand than the `fstring` form